dockerfiles/site/website/compile.sh

```diff
@@ -39,6 +39,8 @@ git checkout $SHA1
 
 # Compilation
 
+export MAVEN_OPTS="-Xmx7168m -Xms2048m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:-UseGCOverheadLimit"
```

Comment:
The docker site works with java6 so I don't know it can be work with maven 3.5. But yes, I will try